rest-servlet/src/main/java/org/hawkular/inventory/rest/RestResources.java

```diff
@@ -51,21 +45,28 @@
 import javax.ws.rs.core.Response;
 import javax.ws.rs.core.UriInfo;
 
-import java.util.Collection;
-
-import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
-import static javax.ws.rs.core.Response.Status.FORBIDDEN;
+import org.hawkular.inventory.api.Environments;
+import org.hawkular.inventory.api.Metrics;
+import org.hawkular.inventory.api.ResolvingToMultiple;
+import org.hawkular.inventory.api.Resources;
+import org.hawkular.inventory.api.filters.Defined;
+import org.hawkular.inventory.api.model.Environment;
+import org.hawkular.inventory.api.model.Feed;
+import org.hawkular.inventory.api.model.Metric;
+import org.hawkular.inventory.api.model.Resource;
+import org.hawkular.inventory.api.model.ResourceType;
+import org.hawkular.inventory.api.paging.Page;
+import org.hawkular.inventory.api.paging.Pager;
+import org.hawkular.inventory.rest.json.ApiError;
 
-import static org.hawkular.inventory.rest.RequestUtil.extractPaging;
-import static org.hawkular.inventory.rest.ResponseUtil.pagedResponse;
 
 /**
  * @author Lukas Krejci
  * @since 1.0
  */
 @Path("/")
-@Produces(value = APPLICATION_JSON)
-@Consumes(value = APPLICATION_JSON)
+@Produces(APPLICATION_JSON)
+@Consumes(WILDCARD)
```

Comment:
good catch, I was trying to fix an issue* with keycloak and forgot on this one.

(*) the api doesn't work if the accept header is not correctly set, that's why our API is not browsable in a browser (browser doesn't set the application/json accept header).